xmodule.grace

```diff
@@ -172,22 +172,22 @@ method checkimport(nm, pathname, isDialect, sourceRange) is confidential {
     }
     util.log 50 verbose "checking module \"{nm}\""
     def gmp = sys.environ.at "GRACE_MODULE_PATH"
-    def pn = filePath.fromString(pathname).setExtension "js"
-    def moduleFileJs = util.file(pn) on (util.outDir)
-                                orPath (gmp) otherwise { l ->
-        def graceFileName = pn.copy.setExtension "grace"
-        def moduleFileGrace = util.file(graceFileName) on(util.outDir)
+    def pnJs = filePath.fromString(pathname).setExtension "js"
+    def pnGrace = pnJs.copy.setExtension "grace"
+
+    var moduleFileJs := util.files([pnJs, pnGrace]) on (util.outDir)
```

Comment:
If you change the role of a variable, please also change its name!  Evidently it wasn't clear, but `moduleFileJs` was intended to always be a JS file.  This code is hard enough to follow without misleading variable names!